### DISCUSSION TITLE
Eero "scans" HomeBridge about 1M times per day

### DISCUSSION THREAD

**Discussion by dmurchie** [iklein99/homebridge-smartthings#84] [Q&A] [answered]
EDIT/SUMMARY/ANSWER: 
Polling of devices by the plugin trigger a "scan" by Eero Secure+. So it would seem that the large number of scans shown in my Eero "Activity" tab is nothing of any particular concern.

------------------------

Didn't want to put this in "Issues" but wondering if anyone has any ideas:

I use Eero Pro6's with their "Secure+" service. Since adding the plugin to HomeBridge, Eero is "scanning" my HomeBridge RPI as many as 1,000,000 times per day (that's 11-12 times per second!). 

For comparison, our laptops (working from home) get scanned about 80,000/week. Before the plugin, all devices combined: 250k scans in a typical week. After plugin, all devices combined: 7M+ scans per week.

To be honest, I'm not sure what exactly triggers the scans (or what exactly Eero is scanning for) but I've seen a few posts suggesting it may be attempts to contact the internet (e.g., "Any time it attempts to connect to the internet utilizing DNS, that'd count as a scan" and "Any time a device connects to something off your network. That’s a scan.").

Any insight into what the homebridge-smartthings-ik plugin might be doing - that HomeBridge wasn't doing before - to trigger so many "scans" by Eero?

TIA...

**Comment by iklein99** [comment, marked as answer]
That is probably due to the plug-in polling all of your devices for updates.  You can control this via settings.   Each device is checked about once ever 10 seconds or so.  You can set the polling time to 0 and stop the polling all together.   Or make it less frequent. Sent from my iPhoneOn Nov 13, 2022, at 9:18 AM, dmurchie ***@***.***> wrote:﻿
Didn't want to put this in "Issues" but wondering if anyone has any ideas:
I use Eero Pro6's with their "Secure+" service. Since adding the plugin to HomeBridge, Eero is "scanning" my HomeBridge RPI as many as 1,000,000 times per day (that's 11-12 times per second!).
For comparison, our laptops (working from home) get scanned about 80,000/week. Before the plugin, all devices combined: 250k scans in a typical week. After plugin, all devices combined: 7M+ scans per week.
To be honest, I'm not sure what exactly triggers the scans (or what exactly Eero is scanning for) but I've seen a few posts suggesting it may be attempts to contact the internet (e.g., "Any time it attempts to connect to the internet utilizing DNS, that'd count as a scan").
Any insight into what the homebridge-smartthings-ik plugin might be doing - that HomeBridge wasn't doing before - to trigger so many "scans" by Eero?
TIA...

**Reply by dmurchie** [reply]
Make sense. Just changed polling to 30 seconds - just to see how that affects "scans" total over the next week. Thanks as always for the good work you're doing...

**Reply by dmurchie** [reply]
Quick update: as a test, I changed polling on all devices to 30.

Sure enough, the number of scans has dropped accordingly.

Thanks again!
